parts/chapters/subsections/12.3/RPTSCHED.fodt

```diff
@@ -5518,7 +5518,7 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
            VeFvHQk2X7L0nYpyLZDNYSnHZ/fE4UJm07Xs1b89ry+9z5H42cPtw+GHdC9XEOSf/BPLYjiQ
            8WqXbj2Yy9dyJE6+fvn3xwv55+5lktv+q4oX/8Qd70rLgs1B0o9U/fntls/WoZ+oJHOJtse2
            y9Vh3b/6u52S+HdU7L+Wp/haxa+wztv71/t65fZL+8y6H8l2VOwZxT5TVz9T/9L1m4oXbWV0
-           XuC8/yb/HovMz3VI+7z+k17m3fsH8f/b8Se+fzu+j6J//LX+8bQr3fkELg6HMugOZJFrqGKC
+           XuC8/yb/HovMz3VI+7z+k17m3fsH8f/b8Se+fzu+j6J//<text:a xlink:href="#__RefHeading___Toc499449_3181922006">LX</text:a>+8bQr3fkELg6HMugOZJFrqGKC
```

Comment:
Looks like if you have an infinite number of images at least one of them will contain a simulation deck :)

Reply:
> Looks like if you have an infinite number of images at least one of them will contain a simulation deck :)

Yes indeed :) great that you found these. I suspect that there could be more like this in other keywords